docs/source/using/profiles.adoc

```diff
@@ -15,7 +15,7 @@ toc::[]
 Each {project} instance can be created with different configurations and users can switch between different instances.
 
 [[create-profile]]
-== Create profile
+== Creating profiles
```

Comment:
Title casing for each heading in this document, please:

"Creating Profiles"
"Listing Existing Profiles"
"Switching Profiles"
"Deleting a Profile"

As a side note, the use of title case seems inconsistent across the documentation. I'll open a separate issue for this.

Reply:
I noticed this as well. In fact I had this already changed in a different branch as well. Will align in this pull request as well.